content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -18,16 +18,16 @@ This task uses the [Bookinfo](/docs/examples/bookinfo/) sample application as th
 
 ## Before you begin
 
-{{< idea >}}
-The following instructions assume you have installed Helm and use it to install Kiali.
-To install Kiali without using Helm, follow the [Kiali installation instructions](https://www.kiali.io/gettingstarted/).
-{{< /idea >}}
+{{< tip >}}
+The following instructions assume you have installed `istioctl` and will use it to install Kiali.
+To install Kiali without `istioctl`, follow the [Kiali installation instructions](https://www.kiali.io/documentation/getting-started/).
+{{< /tip >}}
 
 ### Create a secret
 
-{{< idea >}}
+{{< tip >}}
 If you plan on installing Kiali using the Istio demo profile as described in the [Istio Quick Start Installation Steps](/docs/setup/install/kubernetes/) then a default secret will be created for you with a username of `admin` and passphrase of `admin`. You can therefore skip this section.
-{{< /idea >}}
+{{< /tip >}}
 
```

Comment:
I wonder if it's better to instruct user to create the secret using something like this:
```
kubectl create secret generic -n istio-system kiali --from-literal "username=<USERNAME_HERE>" --from-literal "passphrase=<PASSWORD_HERE>"
```
That would save a lot of space in this page

Reply:
I thought about that. I wondered why we originally wrote it this way - but I think the reason why is to not put the actual credential text in the user's command history. As it is now, the actual username and password is not found in any command line history. If we tell the user to use that one liner, they would have to put their passphrase in the command line and thus putting it in the command line history (which isn't secure). I am guessing that is the reason why this was originally written that way.

Reply:
ok, sounds good, we can improve this later in a follow up.